shared/src/business/useCases/trialSessions/generateNoticesForCaseTrialSessionCalendarInteractor.test.js

```diff
@@ -108,7 +108,7 @@ describe('generateNoticesForCaseTrialSessionCalendarInteractor', () => {
       interactorParamObject,
     );
     expect(
-      applicationContext.getPersistenceGateway().setJobAsProcessing,
+      applicationContext.getPersistenceGateway().setJobStatus,
     ).toHaveBeenCalled();
```

Comment:
Seems like we should also add a test here to indicate that `it should re-attempt the job if it is already processing`. In other words, introduce a test to handle what happens when the job is processing.

Reply:
We created an additional test for reattempted jobs!